corehq/apps/locations/fixtures.py

```diff
@@ -27,16 +29,35 @@ def __contains__(self, item):
         return item in self.by_id
 
 
-def should_sync_locations(last_sync, location_db):
+def fixture_last_modified(user):
```

Comment:
Changed this so that there is now a cached method on `user` which fetches the fixture status that is desired